out/src/js/bestitamazonpay4oxid.js

```diff
@@ -19,6 +19,8 @@ $(document).ready(function() {
         var paymentid = $("input[name=paymentid]:checked").val();
         var $payWithAmazonDiv = $("#payWithAmazonDiv");
 
+        console.log($payWithAmazonDiv);
```

Comment:
Is it necessary to leave a "console.log" in live code?

Reply:
@mknoppik-bestit Gab es bei SW nicht eine ähnliche Situation? Ggfs. einmal zusammen setzen.

Reply:
Can be removed. :)